flocker/node/agents/functional/test_cinder.py

```diff
@@ -198,57 +197,47 @@ class CinderHttpsTests(TestCase):
     """
     Test connections to HTTPS-enabled OpenStack.
     """
-
-    @staticmethod
-    def _authenticates_ok(cinder_client):
-        """
-        Check connection is authorized.
-
-        :return: True if client connected OK, False otherwise.
-        """
+    def setUp(self):
         try:
-            cinder_client.authenticate()
-            return True
-        except Unauthorized:
-            return False
+            self.config = get_blockdevice_config(ProviderType.openstack)
+        except InvalidConfig as e:
+            self.skipTest(str(e))
+        auth_url = self.config['auth_url']
+        if not auth_url.startswith(u"https://"):
+            self.skipTest(
+                "Tests require a TLS auth_url endpoint "
+                "beginning with https://. "
+                "Found auth_url: {}".format(auth_url)
+            )
 
     def test_verify_false(self):
         """
         With the peer_verify field set to False, connection to the
         OpenStack servers always succeeds.
         """
-        try:
-            config = get_blockdevice_config(ProviderType.openstack)
-        except InvalidConfig as e:
-            self.skipTest(str(e))
+        config = self.config.copy()
```

Comment:
Why do we need this .copy? every setUp will recreate the config object, no?
[If not, I'd push the .copy into setUp]